docs/conf.py

```diff
@@ -90,6 +90,13 @@
 html_title = "Stravalib Docs"
 # Theme and css
 html_theme = "furo"
+# Add edit button to furo theme 
```

Comment:
it should ask someone without edit access to the repo to fork the repo if it's not forked. [try it here in the theme documentation itself so you can see how it works.](https://pradyunsg.me/furo/quickstart/) I believe that because you do have write access it will allow you to edit directly. But only you, myself, yihong and hans have write access so i think it should be ok as long as none of us go crazy trying to edit from the edit button :) I can also add protections to the master branch to prevent that from happening by mistake - it will just ask us if we do intend to push directly to master IF we want that.